packages/server-admin-ui/src/views/appstore/appStore.scss

```
@@ -141,7 +141,7 @@
 
   &__grid {
     flex: 1 1 auto;
-    background-color: white;
+    // background-color: var(--sk-card-bg);
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)